newsletter-bundle/src/EventListener/SitemapListener.php

```diff
@@ -68,7 +72,7 @@ public function __invoke(SitemapEvent $event): void
             }
 
             // The target page is protected (see #8416)
-            if ($objParent->protected) {
+            if ($objParent->protected && !$this->security->isGranted(ContaoCorePermissions::MEMBER_IN_GROUPS, $objParent->groups)) {
```

Comment:
This needs to be covered with a test as well - according to the rest

Reply:
Added in aab827f.